db_schema/schema.sql

```diff
@@ -83,6 +83,17 @@ CREATE TABLE spotifyEpisodePerformance (
   PRIMARY KEY (account_id, episode_id)
 );
 
+DROP TABLE IF EXISTS spotifyPodcastMetadata;
+CREATE TABLE spotifyPodcastMetadata (
+  account_id INTEGER NOT NULL,
+  spm_total_episodes INTEGER NOT NULL,
+  spm_starts INTEGER NOT NULL,
+  spm_streams INTEGER NOT NULL,
```

Comment:
is INT sufficient? around 1 billion, I guess it is ok, right?

Reply:
If not we have a luxury problem. 🤑